### ISSUE TITLE
cant install emzed via pip because it requires emzed-optimizations 0.5 which is not on pypi yet

### ISSUE THREAD

**Issue by cfblaeb** [uweschmitt/emzed2#16]
cant install emzed via pip because it requires emzed-optimizations 0.5 which is not on pypi yet

**Comment by uweschmitt** [comment]
Please use http://groups.google.com/group/emzed-users for such issues.

**Comment by cfblaeb** [comment]
Let me rephrase.
Emzed install fails on linux because it asks for emzed-optimization 0.5 which is only available as wheel for macos or windows on pypi. 
Last version of emzed-optimization to include source was 0.38.
To fix, you need to either add source for emzed-optimization 0.5 pypi package or provide another wheel.
